io/latency.md

```diff
@@ -0,0 +1,146 @@
+# Test IO Chaos on TiDB
+## IO Latency
+
+### Description
+Test the availability of TiDB cluster in IO Chaos with IO latency scenarios.
+
+### Hypothesis
+
+When about one third of TiKV pod in TiDB cluster suffered a high IO latency , the TiDB cluster can still provide services normally.
+
+### Preparation
+
+1. Install chaos-mesh
+2. A TiDB cluster on k8s with at least 3 tikv POD.
+3. Runing payload on TiDB cluster
+
+### Quick start
+1. Chaos Mesh fault YAML configuration:
+```
+IOChaos-Latency.yaml
+```
+```
+kind: IOChaos
+apiVersion: chaos-mesh.org/v1alpha1
+metadata:
+  namespace: ${TiDB-Namespace}
+  name: IO-Latency
+spec:
+  selector:
+    namespaces:
+      - ${TiDB-Namespace}
+    pods:
+       ${TiDB-Namespace}:
+        - Some-Of-TiKV-PODS
+  mode: all
+  action: latency
+  delay: 100ms
+  methods:
+    - read
+    - write
+  percent: 100
+  volumePath: /var/lib/tikv
+  containerName: tikv
+```
+2. Using Kubectl to create the experiment:
+```
+kubectl create -f IOChaos-Latency.yaml
+```
+3. Verifying TiDB's status:
+
+    Check QPS & TPS of TiDB in Grafana.
```

Comment:
how about adding a TODO below, we will fill some pictures of Grafana here later.

Reply:
Added.